### PR TITLE
Update squeezebox.markdown for https support

### PR DESCRIPTION
**Pull request by rajlaud**
## Proposed change

Add docs describing https support for Squeezebox.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/95088
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by rajlaud** [comment]
Replaces https://github.com/home-assistant/home-assistant.io/pull/27930 - sorry for the tainted branch and the spam!

File: source/_integrations/squeezebox.markdown [hunk 1]
```diff
@@ -20,7 +20,11 @@ The Squeezebox integration allows you to control a [Logitech Squeezebox](https:/
 {% include integrations/config_flow.md %}
 
 <div class='note'>
-This platform uses the web interface of the Logitech Media Server to send commands. The default port of the web interface is 9000. It is the same port that you use to access the LMS through your web browser. Originally, this platform used the telnet interface, which defaults to 9090. If you previously specified the port in your configuration file, you will likely need to update it.
+This platform uses the web interface of the Logitech Media Server to send commands. The default port of the web interface is 9000. It is the same port that you use to access the LMS through your web browser.
+</div>
+
+<div class='note'>
+The integration now supports Logitech Media Servers behind an https reverse proxy. Please note that Logitech Media Server only natively supports http traffic, so do not select the `https` option unless you have configured such a reverse proxy. Remember you will likely need to update the port from the default as well.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
The integration now supports Logitech Media Servers behind an HTTPS reverse proxy. Please note that Logitech Media Server natively only supports HTTP traffic. Unless you have configured a reverse proxy, do not select the `https` option. If you have configured a reverse proxy, remember to update the port number.
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @rajlaud :+1:

File: source/_integrations/squeezebox.markdown [hunk 2]
```diff
@@ -20,7 +20,11 @@ The Squeezebox integration allows you to control a [Logitech Squeezebox](https:/
 {% include integrations/config_flow.md %}
 
 <div class='note'>
-This platform uses the web interface of the Logitech Media Server to send commands. The default port of the web interface is 9000. It is the same port that you use to access the LMS through your web browser. Originally, this platform used the telnet interface, which defaults to 9090. If you previously specified the port in your configuration file, you will likely need to update it.
+This platform uses the web interface of the Logitech Media Server to send commands. The default port of the web interface is 9000. It is the same port that you use to access the LMS through your web browser.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
This platform uses the web interface of the Logitech Media Server (LMS) to send commands. The default port of the web interface is 9000. It is the same port that you use to access the LMS through your web browser.
```

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @rajlaud

**Pull request merged**